packages/pn-consumers/package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "pagopa-interop-pn-consumers",
+  "private": true,
+  "version": "1.0.0",
+  "description": "PagoPA Interoperability pn-consumers job",
+  "main": "dist",
+  "type": "module",
+  "scripts": {
+    "test": "vitest",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "node --loader ts-node/esm -r 'dotenv-flow/config' --watch ./src/index.ts",
+    "build": "tsc",
+    "check": "tsc --project tsconfig.check.json"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@pagopa/eslint-config": "3.0.0",
+    "@types/node": "20.14.6",
+    "@types/nodemailer": "6.4.9",
+    "pagopa-interop-commons-test": "workspace:*",
+    "@types/uuid": "9.0.8",
+    "uuid": "10.0.0",
+    "prettier": "2.8.8",
+    "testcontainers": "10.9.0",
+    "ts-node": "10.9.2",
+    "typescript": "5.4.5",
+    "mongodb-memory-server": "^8.15.1",
+    "vitest": "1.6.0"
+  },
+  "dependencies": {
+    "@aws-sdk/client-s3": "^3.387.0",
+    "nodemailer": "^6.9.4",
+    "axios": "^1.5.0",
```

let's use fixed versions
```suggestion
    "@aws-sdk/client-s3": "3.387.0",
    "nodemailer": "6.9.4",
    "axios": "1.5.0",
```